pages/proposals/language/English.json

```diff
@@ -6,7 +6,7 @@
     "background": {
       "header": "Welcome to the MozFest 2017 Call for Proposals",
       "body": [
-        "MozFest sessions cover a wide range of topics, but share two important qualities: they are <em>interactive</em> and <em>inclusive</em>.",
+        "MozFest sessions cover a wide range of topics, but share two important qualities: they are <em>interactive</em> and <em>inclusive</em>. We welcome workshops and exhibits, and often shy away from sessions that are exclusively lectures and slide presentations.",
```

Comment:
Let me loop in @Saallen and @KevZawacki here. Hey! The copy we are referring to here is the new sentence that we just added to https://mozillafestival.org/proposals (first paragraph)